.ci.yaml

```diff
@@ -78,7 +78,7 @@ platform_properties:
         ]
       os: Mac-12
       device_type: none
-      xcode: 14c18
+      xcode: 14e222b
```

Comment:
wondering if it's better to use a variable so next time we only need to update one place?